src/corefx/System.Globalization.Native/collation.cpp

```diff
@@ -5,19 +5,25 @@
 //
 
 #include <assert.h>
+#include <pthread.h>
 #include <stdint.h>
+#include <vector>
+#include <map>
 #include <unicode/uchar.h>
 #include <unicode/ucol.h>
 #include <unicode/usearch.h>
 #include <unicode/utf16.h>
 
 const int32_t CompareOptionsIgnoreCase = 1;
-// const int32_t CompareOptionsIgnoreNonSpace = 2;
-// const int32_t CompareOptionsIgnoreSymbols = 4;
-// const int32_t CompareOptionsIgnoreKanaType = 8;
-// const int32_t CompareOptionsIgnoreWidth = 0x10;
+const int32_t CompareOptionsIgnoreNonSpace = 2;
+const int32_t CompareOptionsIgnoreSymbols = 4;
+const int32_t CompareOptionsIgnoreKanaType = 8;
+const int32_t CompareOptionsIgnoreWidth = 0x10;
```

Comment:
a nit, but IMO using 0x across all options would be more straightforward to signify bitwise fields